group-coordinator/src/test/java/org/apache/kafka/coordinator/group/taskassignor/StickyTaskAssignorTest.java

```diff
@@ -403,6 +403,36 @@ public void shouldAssignTasksToClientWithPreviousStandbyTasks() {
         assertEquals(Collections.singleton(0), testMember3.activeTasks().get("test-subtopology"));
     }
 
+    @Test
```

Comment:
This test hits [this](https://github.com/apache/kafka/blob/c5d6f1aab567fe2253187b94aa9270244e230ded/group-coordinator/src/main/java/org/apache/kafka/coordinator/group/taskassignor/StickyTaskAssignor.java#L294) and [this](https://github.com/apache/kafka/blob/c5d6f1aab567fe2253187b94aa9270244e230ded/group-coordinator/src/main/java/org/apache/kafka/coordinator/group/taskassignor/StickyTaskAssignor.java#L226-#L227) (the former fix)